services/src/test/java/edu/unc/lib/dl/cdr/services/rest/modify/ExportCsvIT.java

```diff
@@ -270,17 +273,17 @@ public void exportDeletedResource() throws Exception {
         List<CSVRecord> csvList = parseCsvResponse(response);
 
         assertContainerRecord(csvList, ResourceType.Folder, folderPid, "Folder",
-                FOLDER_PATH, 3, true, 1, false);
+                FOLDER_PATH, 3, true, 1, false, "Staff-only", false);
 
         String pathToWork = FOLDER_PATH + "/" + workPid.getId();
         assertCsvRecord(csvList, ResourceType.Work, workPid, "TestWorkDeleted",
                 pathToWork, 4, true, null, null, null,
-                1, false);
+                1, false, "Restricted", false);
```

Comment:
Seems like the stuff inside the deleted folder should also be staff-only, is it not getting indexed that way?

Reply:
I think it's returning the "actual" permissions not the "effective" permissions.